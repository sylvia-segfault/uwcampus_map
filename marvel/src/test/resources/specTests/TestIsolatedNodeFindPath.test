# Test finding a path on a graph that contains
# an isolated node

LoadGraph graph isolatedNode.tsv
FindPath graph A F