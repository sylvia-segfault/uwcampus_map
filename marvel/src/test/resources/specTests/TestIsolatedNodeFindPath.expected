# Test finding a path on a graph that contains
# an isolated node

loaded graph graph
path from A to F:
A to B via AB
B to D via BD
D to F via DF